File: code-samples/voice/finding-active-calls.php
<?php
// Remember to modify the path ./../ pointing to the location where the RingCentral SDK was installed and the .env file was saved!
require('./../vendor/autoload.php');
$dotenv = Dotenv\Dotenv::createImmutable(__DIR__ . './../');
$dotenv->load();

$rcsdk = new RingCentral\SDK\SDK( $_ENV['RC_APP_CLIENT_ID'],
                                  $_ENV['RC_APP_CLIENT_SECRET'],
                                  $_ENV['RC_SERVER_URL'] );
$platform = $rcsdk->platform();
$platform->login( [ "jwt" => $_ENV['RC_USER_JWT'] ] );

$params = array(
   'view' => 'Simple'
    );
$resp = $platform->get('/account/~/extension/~/active-calls', $params);
foreach ($resp->json()->records as $record) {
    print_r ("Call result: ".$record->result);
}
?>

File: code-samples/voice/finding-active-calls.cs
using System;
using System.Threading.Tasks;
using RingCentral;

namespace Read_User_ActiveCalls
{
  class Program
  {
    static RestClient restClient;
    static async Task Main(string[] args)
    {
      restClient = new RestClient(
      Environment.GetEnvironmentVariable("RC_APP_CLIENT_ID"),
      Environment.GetEnvironmentVariable("RC_APP_CLIENT_SECRET"),
      Environment.GetEnvironmentVariable("RC_SERVER_URL"));
      await restClient.Authorize(Environment.GetEnvironmentVariable("RC_USER_JWT"));
      await read_user_active_calls();
    }
    static private async Task read_user_active_calls()
    {
      var parameters = new ListExtensionActiveCallsParameters();
      parameters.view = "Simple";
      var resp = await restClient.Restapi().Account().Extension().ActiveCalls().Get(parameters);
      foreach (CallLogRecord record in resp.records)
      {
        Console.WriteLine("Call result: " + record.result);
      }
    }
  }
}
